### ISSUE TITLE
TypeError: Browser.getVersion is not a function

### ISSUE THREAD

**Issue by seveibar** [Szpadel/chrome-headless-render-pdf#26]
I'm on Arch Linux, installed via `npm install chrome-headless-render-pdf`. Command doesn't work programmically or via cli, I suspect this was introduced recently, because older versions work fine.

**Comment by seveibar** [comment]
Resolved by reverting to `1.5.0`, that is, via `npm install chrome-headless-render-pdf@1.5.0`

**Comment by seveibar** [comment]
My guess is [this commit](https://github.com/Szpadel/chrome-headless-render-pdf/commit/f605423926fd8f6ab8a480e3a23093fe9a685e81) introduced the bug

**Comment by Szpadel** [comment]
Can you provide more details? How can I reproduce it?

**Comment by seveibar** [comment]
```
npm init
npm install chrome-headless-render-pdf
chmod +x ./node_modules/chrome-headless-render-pdf/cli/chrome-headless-render-pdf.js
./node_modules/chrome-headless-render-pdf/cli/chrome-headless-render-pdf.js --url http://google.com --pdf out.pdf
```

Replicates it with:
Using google-chrome
Waiting for chrome to became available
Connected!
(node:4917) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Browser.getVersion is not a function
(node:4917) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

Chrome version: Google Chrome 60.0.3112.78

**Comment by Szpadel** [comment]
Should be fixed in 1.6.0

**Comment by Szpadel** [comment]
Ps. executable version is installed in `node_modules/.bin/` and it is also available in PATH in npm scripts.
Also you can execute it with yarn from command line: `yarn chrome-headless-render-pdf`
